docs/guides/nrfconnect_factory_data_configuration.md

```diff
@@ -150,6 +151,34 @@ In the factory data set, the following formats are used:
 -   All certificates stored in factory data are provided in the
     [X.509](https://www.itu.int/rec/T-REC-X.509-201910-I/en) format.
 
+#### Appearance field description
+
+The `appearance` field in the factory data set describes the device's visible appearance.
+
+- `finish` - A string name that indicates the visible exterior finish of the product. 
+It refers to the `ProductFinishEnum` enum, and currently, you can choose one of the following names:
+
+|   Name     | Enum value |
+| :--------: | :--------: |
+| `matte`    |     0      |
+| `satin`    |     1      |
+| `polished` |     2      |
+| `rugged`   |     3      |
+| `fabric`   |     4      |
+| `other`    |     255    |
+
+- `primary_color` - A string name that represents the RGB color space of the device's case color, which is the most representative. 
+It refers to the `ColorEnum` enum, and currently, you can choose one of the following names:
+
+(Enum value) color name (`RGB value`)
+
+| (0) $$\color{black} \fcolorbox{white}{white}{black}$$ (`#000000`) | (1) $$\color{#000080}{navy}$$ (`#000080`) | (2) $$\color{#008000}{green}$$ (`#008000`)| (3) $$\color{#008080}{teal}$$ (`#008080`) | (4) $$\color{#800080}{maroon}$$ (`#800080`) |
```

Comment:
Something wrong in the preview:
![image](https://github.com/ArekBalysNordic/connectedhomeip/assets/41291385/b206b84c-275e-4305-bd46-78e155eb5b5b)

Reply:
Repaired.